src/HApp.cpp

```diff
@@ -185,7 +185,7 @@ int main(int argc, char **argv)
 
   return 0;
 }
-
+/*
```

Comment:
In general , I prefer commenting such blocks using `#ifdef DESCRIPTION_OF_CODE` or `#if 0`. This way, those blocks can later be found/identified more easily.